### ISSUE TITLE
Signup: Square-ish gallery images on professional template in style step preview

### ISSUE THREAD

**Issue by iamtakashi** [Automattic/wp-calypso#34143]
If you go through our signup for Professional segment and see the preview in the style step, the images in the gallery is square-ish portrait proportion.

**Comparison** (in Chrome)
<img width="2048" alt="Screen Shot 2019-06-19 at 23 41 28" src="https://user-images.githubusercontent.com/908665/59806322-e21c8300-92eb-11e9-8b2e-20d71054c8cf.png">

The following style that is not in the theme seems to be causing this issue. What's the reason for this to be added for the preview?

```
.wp-block-gallery .blocks-gallery-image figure, .wp-block-gallery .blocks-gallery-item figure {
    flex-direction: column;
    height: auto;
}
```

**Comment by michaeldcain** [comment]
I think this is a result of the fix for #33758.

**Comment by iamtakashi** [comment]
Thanks for the info! I've read the conversation in #33772, and I updated my Chrome, and now the proportion is as expected. Closing.
